packages/info/README.md

```diff
@@ -9,22 +9,55 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag         | Description                   | Type        |     |     |
+| ------------ | ----------------------------- | ----------- | --- | --- |
+| `--json`     | To get the output as JSON     | [ boolean ] |     |     |
+| `--markdown` | To get the output as markdown | [ boolean ] |     |     |
+
+#### Options
+
+| Flag                   | Description                                                    | Type        |
+| ---------------------- | -------------------------------------------------------------- | ----------- |
+| `--help`               | Show help                                                      | [ boolean ] |
+| `--version`            | Show version number                                            | [ boolean ] |
+| `--system` , `--sys`   | System information ( OS, CPU )                                 | [ boolean ] |
+| `--binaries` , `--bin` | Installed binaries                                             | [ boolean ] |
+| `--browsers`           | Installed web browsers                                         | [ boolean ] |
+| `--npmg`               | Globally installed NPM packages ( webpack & webpack-cli only ) | [ boolean ] |
+| `--npmPackages`        | Info about webpack installed in the project                    | [ boolean ] |
```

Comment:
The description is not good. I read it and I don't understand what's the command is about for